addons/website_forum/controllers/main.py

```diff
@@ -73,10 +73,13 @@ def sitemap_forum(env, rule, qs):
                  '''/forum/<model("forum.forum"):forum>/tag/<model("forum.tag"):tag>/questions''',
                  '''/forum/<model("forum.forum"):forum>/tag/<model("forum.tag"):tag>/questions/page/<int:page>''',
                  ], type='http', auth="public", website=True, sitemap=sitemap_forum)
-    def questions(self, forum, tag=None, page=1, filters='all', sorting=None, search='', **post):
+    def questions(self, forum, tag=None, page=1, filters='all', my=None, sorting=None, search='', **post):
         if not forum.can_access_from_current_website():
             raise werkzeug.exceptions.NotFound()
 
+        website_domain = request.website.website_domain()
+        forums = request.env['forum.forum'].search(website_domain)
```

Comment:
if I'm not wrong you just use it to know if you need to display All forums in breadcrumb.

In this case, you can just use search_count instead of search. It will just return the count of row, instead of each id of row.

Reply:
Perfect, thanks!
I may replace it with: 
```python
website_domain = request.website.website_domain()
forums_count = request.env['forum.forum'].search_count(website_domain)
```

There is an issue though... I need to check for this value on each forum page (the button is in the navbar), not only on this template. Should I clone this logic for each controller?